pkg/asset/ignition/bootstrap/baremetal/template.go

```diff
@@ -70,6 +78,16 @@ func GetTemplateData(config *baremetal.Platform, networks []types.MachineNetwork
 	templateData.ProvisioningNetwork = string(config.ProvisioningNetwork)
 	templateData.BaremetalEndpointOverride = fmt.Sprintf("http://%s/v1", net.JoinHostPort(config.APIVIP, "6385"))
 	templateData.BaremetalIntrospectionEndpointOverride = fmt.Sprintf("http://%s/v1", net.JoinHostPort(config.APIVIP, "5050"))
+	templateData.ExternalStaticIP = config.BootstrapExternalStaticIP
+	templateData.ExternalStaticGateway = config.BootstrapExternalStaticGateway
+
+	if config.BootstrapExternalStaticIP != "" {
+		for _, network := range networks {
+			cidr, _ := network.CIDR.Mask.Size()
```

Comment:
Should we be checking that we pick the CIDR from the network that contains the BootstrapExternalStaticIP? In this implementation, we would be picking the 1st network's CIDR. Is that what we want?

Reply:
In a dual stack environment I think you'll always want the external IP to be the first (v4) network, although technically in the DHCP dual stack scenario I guess the NIC could get an IP on both networks

If we ever want IPI to support a case like https://issues.redhat.com/browse/METAL-163 where the BMCs are only accessible via v6, we'd need to revise this for the case where the provisioning network is disabled so the user can specify two IPs.

Considering that a follow-up seems reasonable to me, although we'd end up having to make the install-config interface a list like the networks if that becomes a requirement